slither/solc_parsing/variables/variable_declaration.py

```diff
@@ -47,6 +64,7 @@ def __init__(
         self._initializedNotParsed = None
 
         self._is_compact_ast = False
+        self._from_statement = False
```

Comment:
`_from_statement` may be confusing. How about `_is_implicit_statement` (w/ inverted polarity)?

Reply:
`_from_statement` was tracking if the Variable is local, but I decided to check that it is an instance of `LocalVariable` instead.

Reply:
I see. It makes sense.